src/components/MessageList/VirtualizedMessageList.js

```diff
@@ -101,9 +107,18 @@ const VirtualizedMessageList = ({
       if (message.deleted_at)
         return smartRender(MessageDeleted, { message }, null);
 
-      return <Message message={message} />;
+      return (
+        <Message
+          message={message}
+          groupedByUser={
+            !noGroupByUser &&
+            i > 0 &&
+            message.user.id === messageList[i - 1].user.id
```

Comment:
Can we always assume messages will have users here? I was looking into https://github.com/GetStream/stream-chat-js/blob/a68c23ed1bdbd8dbc71241ec6d7f66b086836a47/src/types.ts#L1366 and I'm not sure.

Reply:
yea the returned messages from API will always have a user AFAIK, the null/undef type in there is not accurate since it's a shared type I guess